test/xpu_api/random/interface_tests/distributions_methods.pass.cpp

```diff
@@ -296,10 +303,41 @@ test(sycl::queue& queue)
 
     // Random number generation
     {
+        {
```

Comment:
I see double brackets. Is it expected?

Reply:
I added them on purpose, but it's my point of view. If they are excessive I will remove them